### PR TITLE
Minor edits to index page

### PR DESCRIPTION
**Pull request by mooncatcrypto**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -485,12 +485,11 @@ <h4 class="title">"Start your cat journey today!"</h4>
 <div class="gutter-vr-15px px row justify-content-between align-items-center text-center text-sm-left">
 <div class="col-lg-4">
 <ul class="social justify-content-center ml-n2">
-<li><a href="#"><em class="social-icon social-icon-transparent fab fa-facebook-f"></em></a></li>
```
Comment (shittydesigns) [inline index.html:488]: Lave it here for now as placeholder?

File: index.html [hunk 2]
```diff
@@ -485,12 +485,11 @@ <h4 class="title">"Start your cat journey today!"</h4>
...
 <li><a href="https://twitter.com/itscattoken"><em class="social-icon social-icon-transparent fab fa-twitter"></em></a></li>
-<li><a href="#"><em class="social-icon social-icon-transparent fab fa-youtube"></em></a></li>
+<li><a href="https://www.youtube.com/watch?v=y3-L3Tg3LPY"><em class="social-icon social-icon-transparent fab fa-youtube"></em></a></li>
```
Comment (shittydesigns) [inline index.html:489]: https://www.youtube.com/watch?v=gOMtsB_WYRU - This would be better I think

**Review by shittydesigns — CHANGES_REQUESTED** [review]

**Comment by mooncatcrypto** [comment]
Sure go ahead!

**Comment by shittydesigns** [comment]
Sure let me have a look it's kinda confusing to look at rn

File: index.html (hunk 1 above)
Comment (shittydesigns) [inline index.html:488]: Leave as placeholder as for now. Maybe... Just maybe there might be a facebook

File: index.html (hunk 2 above)
Comment (shittydesigns) [inline index.html:489]: https://www.youtube.com/watch?v=gOMtsB_WYRU

**Review by shittydesigns — CHANGES_REQUESTED** [review]
All others are fine. Thanks for the contribution!

**Pull request merged**